examples/nextjs-live-cursors-advanced/pages/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { RoomProvider } from "../liveblocks.config";
+import { useRouter } from "next/router";
+import LiveCursors from "../components/LiveCursors";
+import { useMemo, useRef } from "react";
+import styles from "../styles/Index.module.css";
+
+export default function Example() {
+  const roomId = useOverrideRoomId("nextjs-live-cursors-advanced");
+  const cursorPanel = useRef(null);
+
+  return (
+    <RoomProvider
+      id={roomId}
+      /**
+       * Initialize the cursor position to null when joining the room
+       */
+      initialPresence={{
+        cursor: null,
+      }}
+    >
+      <main ref={cursorPanel} className={styles.main}>
+        <LiveCursors cursorPanel={cursorPanel} />
+      </main>
+    </RoomProvider>
+  );
+}
+
+export async function getStaticProps() {
+  const API_KEY = process.env.LIVEBLOCKS_SECRET_KEY;
+  const API_KEY_WARNING = process.env.CODESANDBOX_SSE
+    ? `Add your public key from https://liveblocks.io/dashboard/apikeys as the \`LIVEBLOCKS_SECRET_KEY\` secret in CodeSandbox.\n` +
+      `Learn more: https://github.com/liveblocks/liveblocks/tree/main/examples/nextjs-live-cursors#codesandbox.`
+    : `Create an \`.env.local\` file and add your public key from https://liveblocks.io/dashboard/apikeys as the \`LIVEBLOCKS_SECRET_KEY\` environment variable.\n` +
+      `Learn more: https://github.com/liveblocks/liveblocks/tree/main/examples/nextjs-live-cursors#getting-started.`;
+
+  if (!API_KEY) {
+    console.warn(API_KEY_WARNING);
+  }
+
+  return { props: {} };
+}
```

Comment:
Since this example is using a private key and there's already a warning in `auth.ts` we can remove this.